.github/workflows/test.yml

```diff
@@ -335,25 +336,63 @@ jobs:
         with:
           name: verdaccio-log-${{ matrix.template }}
           path: packages/pwa-kit-create-app/local-npm-repo/verdaccio-windows-${{ matrix.template }}.log
-  lighthouse:
+
+  # NOTE: Commenting out this lighthouse test because it's run against a _local_ server that skews the performance metrics to be better than they really are.
+  # TODO: perhaps run lighthouse in another workflow (e2e one) that deploys to MRT and test against that.
+  # lighthouse:
+  #   needs: changelog-check
+  #   strategy:
+  #     fail-fast: false
+  #   runs-on: ubuntu-latest
+  #   env:
+  #     PROJECT_DIR: generated-retail-react-app-test-project
+  #   steps:
+  #     - name: Checkout
+  #       uses: actions/checkout@v4
+
+  #     - name: Setup Ubuntu Machine
+  #       uses: "./.github/actions/setup_ubuntu"
+  #       with:
+  #         node: ${{ env.MRT_NODE }}
+
+  #     - name: Generate retail-react-app-test-project
+  #       run: |-
+  #         node packages/pwa-kit-create-app/scripts/create-mobify-app-dev.js --outputDir ${{ env.PROJECT_DIR }}
+  #       env:
+  #         GENERATOR_PRESET: retail-react-app-test-project
+  #       timeout-minutes: 8
+
+  #     - name: Run Lighthouse CI on the generated project
+  #       uses: "./.github/actions/lighthouse_ci"
+  #       with:
+  #         project_dir: ${{ env.PROJECT_DIR }}
+  #         config_filename: lighthouserc.retail-react-app.js
+
+  bundle-sizes:
```

Comment:
Running bundle-size related steps in a separate job (rather than within `pwa-kit` job) because if they fail, it's clearer where the failure is. And they won't interfere with the rest of `pwa-kit` job.

Reply:
I think we should have project-specific bundle size limits, not a single global bundle-size test. Since we generate different projects in CI, each with specific presets, every project should have its own custom file size limits.

Reply:
Yup, the `bundle_size_test` action does accept input that allows you to configure the size limits. In this case, the limits are defined as `'{"build/main.js": "10kB", "build/vendor.js": "390kB"}'` (see line 398 below). But we can imagine for other jobs/workflows, the limits could be different.